lib/Runtime/Base/StackProber.cpp

```diff
@@ -63,7 +63,7 @@ StackProber::Initialize()
 #endif
 #else // !_WIN32
     void *lowLimit, *highLimit;
-    ::GetCurrentThreadStackBounds((char**)&lowLimit, (char**)&highLimit);
+    ::GetCurrentThreadStackBounds((char**)&highLimit, (char**)&lowLimit);
```

Comment:
Would it be better to reimplement this function (or even rename it) similar to Windows `GetCurrentThreadStackLimits`, so that all calls are similar?